tethysext/atcore/public/resource_workflows/dataset_workflow_results.css

```diff
@@ -11,6 +11,16 @@
     margin-top: 20px;
 }
 
+table.dataTable > thead .sorting:before, table.dataTable > thead .sorting_asc:before, table.dataTable > thead .sorting_desc:before, table.dataTable > thead .sorting_asc_disabled:before, table.dataTable > thead .sorting_desc_disabled:before {
```

Comment:
The convention is to add a new line after commas to make it more readable like so:

```css
table.dataTable > thead .sorting:before,
table.dataTable > thead .sorting_asc:before,
... {
    right: 1em;
}
```